src/Featurizers/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# ----------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License
+# ----------------------------------------------------------------------
+cmake_minimum_required(VERSION 3.5.0)
+
+macro(Impl)
+    # Defining a function here to introduce a new scope for local variables
+    get_filename_component(_this_path ${CMAKE_CURRENT_LIST_FILE} DIRECTORY)
+
+    include(${_this_path}/cmake/FeaturizersCode.cmake)
```

Comment:
>${_this_path}/ [](start = 12, length = 14)

Can we remove ${_this_path}/  and just do cmake/Featurizers.code ? Anyone can read it.